From 8c894093e9a7495653aac33a9d7fd5605a426071 Mon Sep 17 00:00:00 2001
From: Chintan Patel <RADIXLOCAL\chintan.patel@lnode138.radixweb.local>
Date: Mon, 5 May 2014 16:09:58 +0530
Subject: [PATCH] how updated.

---
 how.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/how.c b/how.c
index 8081b44..cbb54fe 100644
--- a/how.c
+++ b/how.c
@@ -1 +1,9 @@
+
+qwedasa
+fgg
+gf
+f
+f
+
+
 how
-- 
1.8.1.2

